frontend/public/locales/en/apply-adult.json

```diff
@@ -347,6 +347,7 @@
       "characters-valid": "Only letters, numbers and the following characters are accepted: apostrophe ('), comma (,), period (.), hyphen (-) and bracket ()"
     }
   },
+
```

Comment:
Remove that empty line

Reply:
Fixed: 8b2054f7325ac0e04cdca4969c9dc7a800b1673a